src/search/lp/lp_solver.cc

```diff
@@ -37,6 +37,14 @@ using namespace std;
 using utils::ExitCode;
 
 namespace lp {
+#ifndef NDEBUG
+template<class T>
+static bool all_values_are_unique(const vector<T> &v) {
```

Comment:
Move this function next to sorted_unique?

Reply:
I have no strong opinion.

Reply:
You currently also have an additional import here (for `unordered_set`). I think treat both the algorithm and the utils namespace as something that may be included everywhere. But same as Jendrik, I'm also fine either way. It certainly has an advantage to make the module as independent from other modules as possible and we could always make the function more public later on if we need it somewhere else.

Reply:
Indeed, I overlooked that because I first used `set` which didn't need the extra import. ;) 

The disadvantage of defining it locally is that somebody else might define something similar locally unaware of this function here. So I'll just move it to `utils` until somebody has a stronger opinion in a different direction.